homeassistant/components/motion_blinds/entity.py

```diff
@@ -0,0 +1,84 @@
+"""Support for Motion Blinds using their WLAN API."""
+from __future__ import annotations
+
+from typing import TypeVar
+
+from motionblinds import DEVICE_TYPES_GATEWAY, DEVICE_TYPES_WIFI, MotionGateway
+from motionblinds.motion_blinds import MotionBlind
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from . import DataUpdateCoordinatorMotionBlinds
+from .const import ATTR_AVAILABLE, DOMAIN, KEY_GATEWAY, MANUFACTURER
+from .gateway import device_name
+
+_T = TypeVar("_T")
+
+
+class MotionCoordinatorEntity(CoordinatorEntity[DataUpdateCoordinator[_T]]):
```

Comment:
Why not use the DataUpdateCoordinatorMotionBlinds instead of DataUpdateCoordinator[_T]

Reply:
Done, because I kind of did not know exactly what the syntax ment and copied it from another integration......
It is something with typing...